staff_features/shared/step_definitions.rb

```diff
@@ -71,3 +99,11 @@
 Then 'the {string} is checked' do |label|
   expect(page).to have_field(label, checked: true)
 end
+
+Then 'the following error message is displayed' do |messages|
```

Comment:
see my comment in #17

Reply:
I agree that it is more distinguishable that way. I have prepended the word `only` to all the steps that refer to a single message, as well as a check that there is only one message wherever it was missing.

Some notes, which also apply to the other reviews [(16)](https://github.com/archivesspace/e2e-tests/pull/16)

@kdimopulu suggested the same thing before submitting for review, but I thought that since the distinction is already there (plural/singular), it's redundant to prepend the word `only`, to make it as simple and intuitive as possible for the Gherkin writer, without having to search other steps or referring to the documentation.

From the technical point of view, the error/info messages can always be an arbitrary amount of entries in Gherkin/Cucumber, so we could even use the plural version for everything, to make it even simpler.

eg.: Somebody could write the following for a single message, which is totally valid:
```
Given ...
When ...
Then the following error messages are displayed
  | Expression - is required unless a begin or end date is given |
Then ...
```